qualtrics_link/util.py

```diff
@@ -226,10 +226,13 @@ def filter_person_list(person_list):
             employee_list.append(person)
 
     # If more than one Person with Employee role, check if any have prime indicator set to 'Y'
+    # Else return the first employee record
     if len(employee_list) > 0:
```

Comment:
This handles multiple employee records, but not when there is only one employee record being returned.

Essentially, if len(employee_list)  is  0, return employee_list[0] else [contininue on with logic in line 230]